src/Mvc/benchmarkapps/Directory.Build.targets

```diff
@@ -0,0 +1,2 @@
+<Project>
+</Project>
```

Comment:
Add a comment to the empty files about _not_ importing the root `Directory.Build.*` files

Reply:
Done